src/entities/trades/0x/0xTrade.ts

```diff
@@ -179,11 +109,13 @@ export class ZeroXTrade extends TradeWithSwapTransaction {
         ? currencyAmountIn.currency.symbol
         : tokenIn.address
 
+      // slippagePercentage for the 0X API needs to be a value between 0 and 1, others have between 0 and 100
       const response = await fetch(
-        `https://api.0x.org/swap/v1/quote?buyToken=${buyToken}&sellToken=${sellToken}&sellAmount=${
+        `${apiUrl}swap/v1/quote?buyToken=${buyToken}&sellToken=${sellToken}&sellAmount=${
```

Comment:
What do you mean? because in ZEROX_API_URL we have URLs with trailing slash:
`export const ZEROX_API_URL: { [chainId in ChainId]: string } = {
  [ChainId.MAINNET]: 'https://api.0x.org/',
  [ChainId.RINKEBY]: '',
  [ChainId.ARBITRUM_ONE]: '',
  [ChainId.ARBITRUM_RINKEBY]: '',
  [ChainId.XDAI]: '',
  [ChainId.POLYGON]: 'https://polygon.api.0x.org/',
}`